setup.py

```diff
@@ -110,6 +110,7 @@ def run(self):
             'bcipy = bcipy.main:bcipy_main',
             'bcipy-erp-viz = bcipy.helpers.visualization:erp',
             'bcipy-sim = bcipy.simulator:main',
+            'bcipy-sim-gui = bcipy.simulator.ui.gui:main',
```

Comment:
This could be a flag on `bcipy-sim`? I was considering collapsing some of these scripts in the future.

Reply:
I refactored to remove the setup script and include a command line flag instead.